dist/circleci/.circleci/config.yml

```diff
@@ -51,6 +48,26 @@ behat_tests: &behat_tests
     - store_artifacts:
         path: /opt/drupal/artifacts
 
+## Job to run the update path and Cypress tests.
+cypress_tests: &cypress_tests
+  <<: *defaults
+  docker:
+    - image: juampynr/drupal9ci:latest
+    - image: selenium/standalone-chrome-debug:3.7.1-beryllium
```

Comment:
My understanding is that cypress uses its own browser so selenium is not needed, nor chromedriver, nor chrome headless browser.

Reply:
Well spotted, you're right, this is not needed. Probably came from a copy/paste.